cpp/include/Ice/ObjectAdapter.h

```diff
@@ -33,59 +36,49 @@ namespace Ice
         /// @return This object adapter's name.
         [[nodiscard]] virtual const std::string& getName() const noexcept = 0;
 
-        /// Gets the communicator this object adapter belongs to.
+        /// Gets the communicator that created this object adapter.
         /// @return This object adapter's communicator.
         [[nodiscard]] virtual CommunicatorPtr getCommunicator() const noexcept = 0;
 
-        /// Activate all endpoints that belong to this object adapter. After activation, the object adapter can dispatch
-        /// requests received through its endpoints.
+        /// Starts receiving and dispatching requests received over incoming connections.
         /// @see #hold
         /// @see #deactivate
         virtual void activate() = 0;
 
-        /// Temporarily hold receiving and dispatching requests. The object adapter can be reactivated with the
-        /// {@link #activate} operation. <p class="Note"> Holding is not immediate, i.e., after {@link #hold} returns,
-        /// the object adapter might still be active for some time. You can use {@link #waitForHold} to wait until
-        /// holding is complete.
-        /// @see #activate
-        /// @see #deactivate
-        /// @see #waitForHold
+        /// Temporarily holds receiving and dispatching requests. The object adapter can be reactivated with #activate.
```

Comment:
Should we mark hold/waitForHold as deprecated?

We keep them primarily for the tests that use them. We determined there is no reasonable use-cases for these operations; in particular, you can't use them to implement flow-control.

Reply:
We should mention in the remarks that it is only provided for backwards compatibility and it should not be used in new code.

Reply:
Fixed.